config/routes.rb

```diff
@@ -1,9 +1,12 @@
 Rails.application.routes.draw do
-  devise_for :users, :controllers => { registrations: 'registrations' }
+  devise_for :users, controllers: { registrations: 'registrations' }
   # For details on the DSL available within this file, see http://guides.rubyonrails.org/routing.html
 
-  root to: "home#index"
+  root to: 'home#index'
 
   resources :users, only: %i(show index)
 
+  namespace :admin do
+    resources :questions, :documentations, :daily_questions
```

Comment:
That's totally working, but prefer one line by ressource.
This would be particularly useful when we'll have to customise resources like this:

```
resources :questions, only: [:index, :show, :update]
resources :documentations
...
```